### PR TITLE
Clarify personnel security rules

### PR DESCRIPTION
**Pull request by richardTowers**
What
----

The old wording implied that nobody could work for GDS without SC, but we routinely have people join before their clearance comes through. I think it's fair to say (at least for the paas team) that everyone's passed a BPSS check, and we don't give people access to prod until they've got SC.

How to review
-------------

* Check the change is accurate

Who can review
--------------

Not @richardTowers

### CODE REVIEW

**Review by tlwr — APPROVED** [review]

**Review by LeePorte — COMMENTED** [review]
This is for production access to the PaaS not for people who work at GDS. The IA stance on PaaS is that SC is a prerequisite for prod access.

**Comment by richardTowers** [comment]
Do you want to propose an alternate wording @LeePorte? The text prior to this PR definitely reads like it's talking about all GDS people, not just PaaS people with access to production.

File: views/cloud-security-principles.erb [hunk 1]
```diff
@@ -148,7 +148,7 @@
 				<li>GOV.UK PaaS team onboarding</li>
 			</ul>
 
-			<p>GDS follows government and Cabinet Office personnel security processes. All GDS personnel must have at least a security clearance of Security Checked (SC).</p>
+			<p>GDS follows government and Cabinet Office personnel security processes. All GDS personnel must pass a Baseline Personnel Security Standard (BPSS) check. Personnel with access to production systems must have at least a security clearance of Security Checked (SC).</p>
```
Comment (LeePorte) [inline views/cloud-security-principles.erb]:
```suggestion
			<p>GDS follows government and Cabinet Office personnel security processes. All GDS Personnel with access to production systems must have at least a security clearance of Security Checked (SC).</p>
```

Comment (richardTowers) [reply views/cloud-security-principles.erb]: 👍  - should have a lower case `p` though I think.

```suggestion
			<p>GDS follows government and Cabinet Office personnel security processes. All GDS personnel with access to production systems must have at least a security clearance of Security Checked (SC).</p>
```

**Review by tlwr — APPROVED** [review]

**Pull request merged**